data/console/test_node.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -e
+
+test_result_wrapper(){
+  local VERSION=$1
+  local FILE=$2
+
+  local NODE="/usr/bin/node$VERSION"
+  local OUTPUT="/tmp/test_output"
+
+  # Run the test (using common and custom flags if present) and save the output
+  echo "Running $NODE $GLOBAL_FLAGS ${node_flags[$VERSION $FILE]}$FILE"
+  set +e
+  $NODE $GLOBAL_FLAGS ${node_flags[$VERSION $FILE]} $FILE > $OUTPUT 2>&1
+  RESULT=$?
+  set -e
+
+  # If test failed, keep track of failure and print out its output
+  if [ $RESULT -ne 0 ]; then
+    TEST_RESULT="failed"
+    echo "Node v$VERSION Test $FILE" >> $FAILED_TEST_LIST
+
+    echo "FAILED"
+    echo "Test Output:"
+    cat $OUTPUT
+  else
+    echo "OK"
+  fi
+  rm $OUTPUT
+}
+
+test_node_version(){
+  local VERSION="$1"
+
+  echo "Starting to test node version $VERSION"
+
+  # Get latest sources to make sure tests contain correct patches
+  zypper -n --no-gpg-checks si --repo node_sources -D "nodejs$VERSION" #> /dev/null 2>&1
+
+  local SOURCE_FILE=""
+  SOURCE_FILE=$(cd /usr/src/packages/SOURCES/ && find . -type f -iname "node-v$VERSION*.tar.xz")
+  SOURCE_FILE=${SOURCE_FILE:2} # strip ./ from name
+
+  if [ -z "$SOURCE_FILE" ]
```

Comment:
Sometimes  is condition in one line sometimes in multiple lines, try to use always same style.

Reply:
fixed!